components/cam/src/physics/cam/flux_avg.F90

```diff
@@ -180,8 +181,11 @@ subroutine smooth(new, old, res, nstep, deltat, ncol)
    integer,  intent(in)    :: ncol
 
    real(r8) :: temp(pcols)
+   real(r8) :: maxres
    integer i
 
+   real(r8), parameter :: timescale = 14400._r8    ! 4 hours
```

Comment:
Should this be a namelist parameter?

Reply:
yea, but we don't need this for pg2, so it's not a priority. I would argue we should do this in a small future PR.

Reply:
Right, just noting.